src/Symfony/Bundle/SecurityBundle/Tests/Security/FirewallMapTest.php

```diff
@@ -88,7 +88,7 @@ public function testGetListeners(Request $request, bool $expectedState)
         $this->assertEquals([[$listener], $exceptionListener, $logoutListener], $firewallMap->getListeners($request));
         $this->assertEquals($firewallConfig, $firewallMap->getFirewallConfig($request));
         $this->assertEquals('security.firewall.map.context.foo', $request->attributes->get(self::ATTRIBUTE_FIREWALL_CONTEXT));
-        $this->assertEquals($expectedState, $request->attributes->get('_stateless'));
+        $this->assertEquals($expectedState, $request->isStateless());
```

Comment:
```suggestion
        $this->assertEquals($expectedState, method_exists($request, 'isStateless') ? $request->isStateless() : $request->attributes->get('_stateless'));
```

Reply:
I bump the dependency instead https://github.com/symfony/symfony/pull/57852/commits/6e41e0990a6af5e176c9d565c25ec348458581d9 to be consistent with other changes.

Reply:
I would not bump the requirement here as the only change for which that would be necessary is a test.

Reply:
Changed in https://github.com/symfony/symfony/pull/57852/commits/2fae617942d2cfef49e4a40967cb560417a0d39e

Reply:
Rechanged later since it's now used in src too.